libraries/cli/include/cli/default_config.hpp

```diff
@@ -90,6 +90,10 @@ const char *default_json = R"foo({
           {
             "name": "SUMMARY",
             "verbosity": "INFO"
+          },
+          {
```

Comment:
it should not be part of default as it is just for testing

Reply:
it is not testing - it is statistics collection, which would be definitely worth to see also on mainnet and compare it with other networks

Reply:
ok, then this can not be part of console output at all, but as we discussed just file output, even for testnet. And maybe we should increase logging interval to few minutes, because 30s does not show anything

Reply:
I see it as a separate log file just with that data. So we can easily get it from users. If we will add new configuration(`logging.configurations`) with different output it should work I believe

Reply:
hmm now when I look at it maybe it should not be logged at all and we will use simply rpc :) Just so you understand how this works. For example the node is running for 1 day, during this day I collect packets stats every 30 sec and I keep track of max values(max count/max size) for each packet per these 30 sec time periods. So if the node is running 1 day and I call rpc, I get the max from this 1 day, if it is running for 1 week and call rpc, I get max from this week. When I think about it we dont need to log this regularly as for example if we collect max stats (in terms of max size) at the node startup when it starts syncing, this max stats will be printed in every log after that and it wont change

Reply:
@MatusKysel not sure if it is more clear now, but it does not matter that much if you use 30 sec, 5 sec or 2 minute period.... you simply need some period which will be then used also for bandwidth limits

Reply:
Ok then, I am voting for RPC call